test/loginNewUser.js

```diff
@@ -0,0 +1,17 @@
+const request = require('supertest');
+const app = require('../app');
+
+const loginNewUser = async loginUser => {
+  await request(app)
+    .post('/users')
+    .send(loginUser);
+  const { body } = await request(app)
+    .post('/users/sessions')
+    .send({
+      email: loginUser.email,
+      password: loginUser.password
+    });
+  return body;
+};
+
+module.exports = loginNewUser;
```

Comment:
Nice, but I recommend renaming this file to something more generic and put it in a utils folder, something like this `test/utils/users`